src/fullsignalk.js

```diff
@@ -177,72 +225,100 @@ function handleOtherSource(sourceLeaf, source, timestamp) {
 }
 
 function addValues(context, contextPath, source, timestamp, pathValues) {
-  var len = pathValues.length;
-  for (var i = 0; i < len; ++i) {
-    addValue(context, contextPath, source, timestamp, pathValues[i]);
-  }
+  pathValues.forEach(function(pathValue) {
```

Comment:
An arrow function would be a bit more readable I think.

Reply:
OK